janus_server/src/aggregator/aggregation_job_driver.rs

```diff
@@ -1324,14 +1376,19 @@ mod tests {
                         )
                         .await?
                         .unwrap();
-                    Ok((aggregation_job, report_aggregation))
+                    let batch_unit_aggregations = tx
+                        .get_batch_unit_aggregations_for_task_in_interval::<PRIO3_AES128_VERIFY_KEY_LENGTH, Prio3Aes128Count>(task_id, Interval::new(nonce.time().to_batch_unit_interval_start(task.min_batch_duration).unwrap(), task.min_batch_duration).unwrap(), &())
```

Comment:
I'm surprised `rustfmt` doesn't wrap this line. Could you please try to confine it to 100 columns or so?

Reply:
Yep, fixed. I line-broke some macro invocations too. (I'm not sure why it didn't apply TBH, it skips macro invocations sometimes but I've never seen it skip a non-macro function call...)

Reply:
rustfmt is significantly more lax inside macro invocations, as it is not aware of that macro's DSL. For example, if it wraps an `assert_eq!()` onto multiple lines, the whitespace difference is safe, as it's no different at the token level, but rustfmt doesn't know whether the macro would be tolerant of adding a trailing comma to the last argument, now that it's on its own line.

I have seen rustfmt fail to act on normal code in certain places (somewhere in datastore.rs, I think) but I don't have any explanation for why. It could be we've reached some intentional nesting/complexity limit, or it could be a bug possibly. If so, minimizing a test case would be a bear.